packages/tools/devtools/devtools-core/src/data-visualization/DefaultEditors.ts

```diff
@@ -12,19 +12,37 @@ import { SharedCounter } from "@fluidframework/counter";
 import { SharedString } from "@fluidframework/sequence";
 
 import { ISharedObject } from "@fluidframework/shared-object-base";
+import { SharedCell } from "@fluidframework/cell";
```

Comment:
Nit: Can we use `ISharedCell` here instead of `SharedCell`?

Reply:
Ideally, we would be using the interfaces instead of the classes for the others too, but we don't need to do that in this PR.

Reply:
To double check, is defaultVisualizers current using the interfaces?

Reply:
Oh, actually we have to use the classes for the `.getFactory()` calls at the bottom of the file. So, we really do need the classes in this file 😕